pycbc/types/timeseries.py

```diff
@@ -203,7 +203,7 @@ def at_time(self, time, nearest_sample=False):
         """
         if nearest_sample:
             time += self.delta_t / 2.0
-        return self[int((time-self.start_time)*self.sample_rate)]
+        return self[int(round((time-self.start_time)*self.sample_rate))]
```

Comment:
Is this issue related to the SNR series delta_t being different?

Anyway, I think you can achieve the same result by calling `at_time` with `nearest_sample=True`, without having to do this change.

Reply:
ah! I didn't use `nearest_sample` before assert. It is working fine.